Content/paket.dependencies

```diff
@@ -44,8 +46,8 @@ group Client
     clitool dotnet-fable ~> 2
 
 group Build
-    storage: none
     source https://api.nuget.org/v3/index.json
+    storage: none
```

Comment:
question to @matthid : can fake 5 dependencies be also restricted to a certain framework? e.g. `framework: netstandard2.0` ?

Reply:
yes, in fact when using "paket-inline dependencies" `framework: netstandard2.0` will be added by default:

https://github.com/fsharp/FAKE/blob/25a04f35a00d0fe892523d992f6245edeb9e0eca/src/app/Fake.Runtime/FakeRuntime.fs#L118-L119